app/User.php

```diff
@@ -556,4 +556,9 @@ public function isWordsmith(): bool
     {
         return $this->hasRole('wordsmith');
     }
+      // A User hasOne Google2FA
+    public function PasswordSecurity()
```

Comment:
Kanka uses camelCase in the app for function names, so please use `passwordSecurity()` instead